bin/serverless

```diff
@@ -7,10 +7,9 @@ let argv = require('minimist')(process.argv.slice(2));
 if (argv.debug) process.env.DEBUG = '*';
 
 let S           = require('../lib/Serverless'),
-    SUtils      = require('../lib/utils/index'),
-    serverless  = new S({
-      interactive: typeof process.env.CI !== 'undefined' ? false : true,
-      projectPath: SUtils.getProjectPath(process.cwd())
+    Project     = require('../lib/ServerlessProject'),
```

Comment:
I was thinking a lot about merging Project and Serverless together, but at the end, it doesnt seem to be right. Like, if you will ever get that web UI so serverless will be a daemon running and user will want to control his projects over http, and there will be few projects (you're encouraging small exchangable parts!), separate classes with clear responsibilities will make that happen for free. And, more importantly, Serverless class does have responsibility not really belonging to the project (e.g. pipeline execution).

Now, providing convenience method to accept path and calling finding function inside Serverless constructor is some option, indeed. Personally I dont like it, even if it will make programmatic usage a bit more verbose (but more flexible and clean too!), but I see your point.

The thing I really dont like about the previous architecture is overwhelming number of object literals, and everything being rooted into specific properties + parsing json files on the fly. That hurts the code a lot, and make every change harder (because, if you want to update some functionality across codebase, you have to change 50 places instead of 1, and your chance to get into trouble skyrockets).

Anyway, having said all of the above, I'll update the code, so Serverless constructor will accept path or string or undefined as first argument, and will try to find project under the hoods, if string is given.